turbopack/crates/turbo-tasks-backend/src/backend/storage.rs

```diff
@@ -403,15 +407,45 @@ macro_rules! get {
     };
 }
 
+macro_rules! get_mut {
+    ($task:ident, $key:ident $input:tt) => {
+        if let Some($crate::data::CachedDataItemValue::$key { value }) = $task.get_mut(&$crate::data::CachedDataItemKey::$key $input).as_mut() {
+            let () = $crate::data::allow_mut_access::$key;
+            Some(value)
+        } else {
+            None
+        }
+    };
+    ($task:ident, $key:ident) => {
+        $crate::backend::storage::get_mut!($task, $key {})
+    };
+}
+
 macro_rules! iter_many {
-    ($task:ident, $key:ident $input:tt => $value:ident) => {
+    ($task:ident, $key:ident $input:tt => $value:expr) => {
         $task
             .iter($crate::data::indicies::$key)
             .filter_map(|(key, _)| match *key {
                 $crate::data::CachedDataItemKey::$key $input => Some($value),
                 _ => None,
             })
     };
+    ($task:ident, $key:ident $input:tt => $value:expr) => {
```

Comment:
This branch will never be matched? The pattern is exactly the same as the branch above it (though this one is missing a deref).

Should be fixed in #71061